CRM/Contact/BAO/Query.php

```diff
@@ -6257,16 +6259,14 @@ public static function processSpecialFormValue(&$formValues, $specialFields, $ch
    *
    * @param string|CRM_Utils_Sort $sort
    *   The order by string.
-   * @param bool $sortByChar
-   *   If true returns the distinct array of first characters for search results.
    * @param null $sortOrder
    *   Who knows? Hu knows. He who knows Hu knows who.
    * @param string $additionalFromClause
    *   Should be clause with proper joins, effective to reduce where clause load.
    * @return array
    *   list(string $orderByClause, string $additionalFromClause).
    */
-  protected function prepareOrderBy($sort, $sortByChar, $sortOrder, $additionalFromClause) {
+  protected function prepareOrderBy($sort, $sortOrder, $additionalFromClause) {
```

Comment:
additionalFromClause is no longer required in this signature - I haven't removed to keep this from going stale but we should do as a follow up